### PR TITLE
feat: add instance.UpdateSecurityGroupRule 

### PR DESCRIPTION
**Pull request by jerome-quere**

### CODE REVIEW

File: api/instance/v1/security_group_utils.go [hunk 1]
```diff
@@ -90,3 +90,101 @@ func (s *API) UpdateSecurityGroup(req *UpdateSecurityGroupRequest, opts ...scw.R
...
+
+type UpdateSecurityGroupRuleResponse struct {
+	Rule *SecurityGroupRule `json:"security_rule"`
+}
+
+// UpdateSecurityGroupRule updates a security group.
+func (s *API) UpdateSecurityGroupRule(req *UpdateSecurityGroupRuleRequest, opts ...scw.RequestOption) (*UpdateSecurityGroupRuleResponse, error) {
+	var err error
+
+	if req.Zone == "" {
```
Comment (QuentinBrosse) [inline api/instance/v1/security_group_utils.go]: nit: `fmt.Sprint(req.Zone) == ""` (just for consistency ?)

File: api/instance/v1/security_group_utils_test.go [hunk 2]
```diff
@@ -62,3 +62,115 @@ func TestAPI_UpdateSecurityGroup(t *testing.T) {
...
+
+	instanceAPI := NewAPI(client)
+
+	var (
+		zone         = scw.ZoneFrPar1
+		organization = "b3ba839a-dcf2-4b0a-ac81-fc32370052a0"
+	)
+
+	bootstrap := func(t *testing.T) (*SecurityGroup, *SecurityGroupRule, func()) {
+
```
Comment (QuentinBrosse) [inline api/instance/v1/security_group_utils_test.go]: nit: newline

Comment (jerome-quere) [reply api/instance/v1/security_group_utils_test.go]: We always start a func with a new line no ?

Comment (QuentinBrosse) [reply api/instance/v1/security_group_utils_test.go]: [nope](https://github.com/scaleway/scaleway-sdk-go/pull/128/files#diff-1e71dd53fd89dedc69a5a17bc3efb9a1R114)

**Pull request merged**